mail_server/models/__init__.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from coaster.sqlalchemy import IdMixin, TimestampMixin, BaseMixin, BaseNameMixin, JsonDict
+from coaster.db import db
+from coaster.utils import LabeledEnum
+from flask_lastuser.sqlalchemy import UserBase2, ProfileBase
+from .user import *
+from baseframe import __
+
+
+class Organization(ProfileBase, db.Model):
+    __tablename__ = 'organization'
```

Comment:
We don't need this model at all. This project can be limited to `siteadmin`-based access.